From: Tommy Webb <tommy@calyxinstitute.org>
Date: Mon, 10 Feb 2025 22:57:04 +0000
Subject: Fix build after "Restore chrome password store"

Remove a now-unused JNI function as it was leading to build failures
because of -Werror,-Wunused-function. Ordinarily we could work around
this with a well-placed #pragma line, but this error occurred inside
of generated JNI headers, leaving no clear place to change this.

Path was: gen/jni_headers/chrome/browser/autofill/android/jni_headers/
AutofillClientProviderUtils_jni.h

Change-Id: I9f4d48520043694e3f2f98937397a31cada5d854
---
 .../autofill/AutofillClientProviderUtils.java | 34 -------------------
 1 file changed, 34 deletions(-)

diff --git a/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java b/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java
--- a/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java
+++ b/chrome/browser/autofill/android/java/src/org/chromium/chrome/browser/autofill/AutofillClientProviderUtils.java
@@ -95,39 +95,5 @@ public class AutofillClientProviderUtils {
         return AndroidAutofillAvailabilityStatus.AVAILABLE;
     }
 
-    @CalledByNative
-    public static void setThirdPartyModePref(boolean usesPlatformAutofill) {
-        Editor editor =
-                ContextUtils.getApplicationContext()
-                        .getSharedPreferences(
-                                AUTOFILL_THIRD_PARTY_MODE_SHARED_PREFS_FILE, Context.MODE_PRIVATE)
-                        .edit();
-        editor.putBoolean(AUTOFILL_THIRD_PARTY_MODE_KEY, usesPlatformAutofill);
-        editor.apply();
-    }
-
-    @CalledByNative
-    public static void unsetThirdPartyModePref() {
-        Editor editor =
-                ContextUtils.getApplicationContext()
-                        .getSharedPreferences(
-                                AUTOFILL_THIRD_PARTY_MODE_SHARED_PREFS_FILE, Context.MODE_PRIVATE)
-                        .edit();
-        editor.remove(AUTOFILL_THIRD_PARTY_MODE_KEY);
-        editor.apply();
-    }
-
-    @CalledByNative
-    public static void setAutofillOptionsDeepLinkPref(boolean featureOn) {
-        featureOn = false;
-        Editor editor =
-                ContextUtils.getApplicationContext()
-                        .getSharedPreferences(
-                                AUTOFILL_OPTIONS_DEEP_LINK_SHARED_PREFS_FILE, Context.MODE_PRIVATE)
-                        .edit();
-        editor.putBoolean(AUTOFILL_OPTIONS_DEEP_LINK_FEATURE_KEY, featureOn);
-        editor.apply();
-    }
-
     private AutofillClientProviderUtils() {}
 }
-- 

